openlibrary/templates/account/create.html

```diff
@@ -12,12 +12,67 @@ <h1>$_("Sign Up")</h1>
 </div>
 
 <div id="contentBody">
-
 <script type="text/javascript">
 window.q.push( function(){
```

Comment:
Move this JS to a new file in openlibrary/plugins/openlibrary/js/ and then load it conditionally when #username is present in the DOM. See the existing line `if (document.getElementsByClassName('editions-table--progressively-enhanced').length) {` for an example.

It will likely help to move this code first before pushing ahead with the feature changes.

Reply:
Done :)